src/main/java/htsjdk/samtools/SAMFileWriterFactory.java

```diff
@@ -89,6 +92,18 @@ public SAMFileWriterFactory setCreateMd5File(final boolean createMd5File) {
         return this;
     }
 
+    /**
+     * Set the deflater factory used by BAM writers created by this writer factory. Must not be null.
+     * If this method is not called, the default  {@link DeflaterFactory} is used which creates the default JDK {@link Deflater}.
+     * This method returns the factory itself. */
```

Comment:
`returns the factory` -> `returns the SAMFileWriterFactory` (since there are two factories involved here)